homeworks/egetoimre/3_conditionals_loops/ticket_booking.py

```diff
@@ -0,0 +1,69 @@
+favorite_films = ["Keresztapa", "Nagymenők", "Ghostbusters", "Star Wars", "Kémek a sasfészekben", "Oroszországból szeretettel"]
+
+while True:
+    chosen_film = input("Melyik filmre szeretne jegyet venni?")
+    if chosen_film in favorite_films:
+        break
+    else:
+        print("Jelenleg ezeket a filmeket vetítjük. Kérem ezek közül válasszon:")
+        for movie in favorite_films:
+            print(movie)
+
+reserved_seats_sum = 0
+seats_in_the_cinema = 25
+
+seats = [
+    [0,0,0,0,0],
+    [0,0,0,0,0],
+    [0,0,0,0,0],
+    [0,0,0,0,0],
+    [0,0,0,0,0],
+]
+
+print("Szabad helyek a teremben:")
+
+for rows in seats:
+    print(rows)
+
+while True:
+
+    ticket_total = int(input("Hány jegyet szeretne vásárolni?"))
+
+    if ticket_total == 0:
+        print("Legalább 1 jegy vásárlása kötelező!")
+
+    for reserved_seats in seats:
```

Comment:
Ezt nem szükséges a while loop-ba tenni, megteheted előtte is akár 1 soron:

`reserved_seats_sum = sum(seat for row in seats for seat in row)`

vagy persze a nested for loop-pal amit írtál, ezáltal egyszerűsödne a while loop-od tartalma is:

```
while True:
    ticket_total = int(input("Hány jegyet szeretne vásárolni?"))

    if ticket_total == 0:
        print("Legalább 1 jegy vásárlása kötelező!")
    elif (reserved_seats_sum + ticket_total) <= seats_in_the_cinema:
        break
    else:
        print("Nincs elég hely a teremben. Kérem módosítsa a jegyek darabszámát!")
```